architecture/index.adoc

```diff
@@ -148,90 +151,92 @@ manage secrets for pulling and pushing images, builds, and the deployment
 components, and also allows application code to easily leverage those secrets.
 endif::[]
 
-All communication channels with the REST API as well as between
-xref:infrastructure_components/kubernetes_infrastructure.adoc#master-components[master components]
-such as etcd and API server, are secured with
-link:https://en.wikipedia.org/wiki/Transport_Layer_Security[TLS]. TLS
-provides strong encryption, data integrity, and authentication of servers with
-X.509 server certificates and public key infrastructure.
+[[arch-index-how-is-it-secured-tls]]
+=== TLS Support
+
+All communication channels with the REST API, as well as between
+xref:infrastructure_components/kubernetes_infrastructure.adoc#master-components[master components] such as etcd and the API server, are secured with TLS. TLS provides
+strong encryption, data integrity, and authentication of servers with X.509
+server certificates and public key infrastructure.
 ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
-By default, a new internal PKI is created for each deployment of {product-title}.
-xref:../install_config/certificate_customization.adoc#install-config-certificate-customization[Custom certificates]
-for public hosts are supported as well. The internal PKI uses 2048 bit RSA keys
-and SHA-256 signatures.
+By default, a new internal PKI is created for each deployment of
+{product-title}. The internal PKI uses 2048 bit RSA keys and SHA-256 signatures.
+endif::[]
+ifdef::openshift-origin,openshift-enterprise[]
+xref:../install_config/certificate_customization.adoc#install-config-certificate-customization[Custom certificates] for public hosts are supported as well. 
 endif::[]
 
-{product-title} uses Golang’s standard library implementation of crypto/tls and
-does not depend on any external crypto and TLS libraries link:https://golang.org/pkg/crypto/tls/[crypto/tls].
-Additionally the client depends on external libraries for GSSAPI authentication
-and OpenPGP signatures. GSSAPI is typically provided by either
-MIT Kerberos or Heimdal Kerberos, which both use OpenSSL's libcrypto. OpenPGP
-signature verification is handled by libgpgme and GnuPG.
-
-The insecure versions SSL 2.0 and SSL 3.0 are unsupported and not available.
-{product-title} server and `oc` client only provide TLS 1.2 by default. TLS
-1.0 and TLS 1.1 can be enabled in the server configuration. Both server and
-client prefer modern cipher suites with authenticated encryption algorithms
-and perfect forward secrecy. Cipher suites with deprecated and insecure algorithms
-such as RC4, 3DES, and MD5 are disabled. Some internal clients (e.g. LDAP authentication)
-have less restrict settings with TLS 1.0 to 1.2 and more cipher suites enabled.
-
-.Supported TLS versions
+{product-title} uses Golang’s standard library implementation of
+link:https://golang.org/pkg/crypto/tls/[*crypto/tls*] and does not depend on any
+external crypto and TLS libraries. Additionally, the client depends on external
```

Comment:
What is the `client` here? The `oc` client?

Reply:
Yes